release-tools/build.make

```diff
@@ -275,3 +275,18 @@ test-shellcheck:
 .PHONY: check-go-version-%
 check-go-version-%:
 	./release-tools/verify-go-version.sh "$*"
+
+# Test for spelling errors.
+.PHONY: test-spelling
+test: test-spelling
+test-spelling:
+	@ echo; echo "### $@:"
+	@ ./release-tools/verify-spelling.sh
```

Comment:
After the change discussed in https://github.com/kubernetes-csi/csi-release-tools/pull/124#pullrequestreview-565256413 this invocation needs the absolute path of the directory which is to be checked here.

I didn't know that we were enabling this for *all* repos. This part here wasn't in PR that I reviewed (https://github.com/kubernetes-csi/csi-release-tools/pull/124/), so I missed this.

Is that really what we want? It will cause a lot of work the next time someone tries to update csi-release-tools in a repo because in addition to updating csi-release-tools, also all new errors found by this check will have to be fixed.

I would prefer to keep this check optional at first, i.e. *not* make it a dependency of `test` in `build.make`. If you want to activate it in csi-driver-nfs, you can put that dependencies into your top-level Makefile.

@msau42 what do you think?

Reply:
In addition to passing `"$(pwd)"` here (for the sake of clarity) it would be good to also make the default root an absolute path (in case someone invokes the script directly without parameters).

Reply:
I think we should have it enabled in all repos, but making it optional so we can roll it out more gradually is fine.

Reply:
Done!